SampleApp/build.gradle

```diff
@@ -54,6 +62,11 @@ dependencies {
     } else {
         compile 'io.card:android-sdk:REPLACE_VERSION'
     }
+
+    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.4'
+
+    androidTestCompile "com.android.support.test.espresso:espresso-core:2.2.2"
+    androidTestCompile "com.github.lkorth:device-automator:01d85912ec"
```

Comment:
I haven't released a tagged version of it yet (https://github.com/lkorth/device-automator/tree/01d85912ec9466db465389a08d2e1df9814a831c#download-device-automator), but soon!